ecosystem/typescript/sdk/src/providers/indexer.ts

```diff
@@ -79,10 +126,187 @@ export class IndexerClient {
    * @param idHash token id hash
    * @returns GetTokenActivitiesQuery response type
    */
-  async getTokenActivities(idHash: string): Promise<GetTokenActivitiesQuery> {
+  async getTokenActivities(idHash: string, options?: PaginationArgs): Promise<GetTokenActivitiesQuery> {
     const graphqlQuery = {
       query: GetTokenActivities,
-      variables: { idHash },
+      variables: { idHash, offset: options?.offset, limit: options?.limit },
+    };
+    return this.queryIndexer(graphqlQuery);
+  }
+
+  /**
+   * Queries an account coin data
+   *
+   * @param ownerAddress Owner address
+   * @returns GetAccountCoinsDataQuery response type
+   */
+  async getAccountCoinsData(ownerAddress: MaybeHexString, options?: PaginationArgs): Promise<GetAccountCoinsDataQuery> {
+    const address = HexString.ensure(ownerAddress).hex();
+    IndexerClient.validateAddress(address);
+    const graphqlQuery = {
+      query: GetAccountCoinsData,
+      variables: { owner_address: address, offset: options?.offset, limit: options?.limit },
+    };
+    return this.queryIndexer(graphqlQuery);
+  }
+
+  /**
+   * Queries an account token acount
+   *
+   * @param ownerAddress Owner address
+   * @returns AccountTokensCountQuery response type
+   */
+  async getAccountTokensCount(ownerAddress: MaybeHexString): Promise<GetAccountTokensCountQuery> {
+    const address = HexString.ensure(ownerAddress).hex();
+    IndexerClient.validateAddress(address);
+    const graphqlQuery = {
+      query: GetAccountTokensCount,
+      variables: { owner_address: address },
+    };
+    return this.queryIndexer(graphqlQuery);
+  }
+
+  /**
+   * Queries an account transactions acount
```

Comment:
```suggestion
   * Queries an account transactions count
```

Same thing for other comments, `acount` -> `count`.

Reply:
Though something like "Gets the count of transactions submitted by an account" would be more descriptive.